scripts/checks.sh

```diff
@@ -456,9 +456,9 @@ function check_image_corruption()
 		do
 			# TODO: the timeout here is probably too short to be effective in most cases, but let's gather
 			# data and change if necessary
-			if ! ${TIMEOUT_CMD} ${ENG} save "${i}" > /dev/null ; then
+			if ! ${TIMEOUT_CMD} ${ENG} save "${i}" > /dev/null 2>&1; then
 				# this retval indicates a timeout
-				if (( $? = 124 )); then
+				if [[ "$?" -eq 124 ]]; then
```

Comment:
Out of curiosity, why the change from `(( ))` to `[[ ]]`?  My understanding is the former is meant for arithmetic; is it because we're not *really* doing arithmetic here, but rather checking a return value (which just happens to be expressible as arithmetic, but doesn't really capture what we intend)?